test/powershell/engine/Basic/StandardLibraryTypes.Tests.ps1

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This is a simple type check to validate that types in PowerShellStandard are present in System.Management.Automation.dll
+# It does not check member presence and info, just the properties of the type
+Describe "Types referenced by PowerShell Standard should not be missing" {
+    BeforeAll {
+        $assets = [io.path]::Combine("$PSScriptRoot", "assets", "standardtypes.csv")
+        # The properties of a type which should match PowerShell Standard
+        # These are not members of the type
+        $typeProperties = "IsCollectible", "IsSZArray", "IsByRefLike", "IsConstructedGenericType", "IsGenericType", "IsGenericTypeDefinition",
```

Comment:
Would be nicer to put these one-per-line for diffing/editing

Reply:
done!